db/internal_stats.cc

```diff
@@ -1738,7 +1738,7 @@ void InternalStats::DumpCFMapStats(
   assert(vstorage);
 
   int num_levels_to_check =
-      (cfd_->ioptions()->compaction_style != kCompactionStyleFIFO)
+      (cfd_->ioptions()->compaction_style == kCompactionStyleLevel)
           ? vstorage->num_levels() - 1
           : 1;
 
```

Comment:
Compaction score is only calculated for L0 for universal compaction and vstorage->CompactionScoreLevel(i) returns 0 always. So compaction_score in the loop below overwrites L0's compaction score to 0.